docs/pages/upgrade.mdx

```diff
@@ -6,6 +6,12 @@ Version 5: Rework of our foundations
 
 ### Theme
 
+#### Modal
+
+- We have adjust colors and spacing.
+- **Modal.Title** is now named **Modal.Header** and have props `title` (mandatory) and `subtitle` (optional).
+- **Modal.Cover** has been removed.
+
```

Comment:
Sorry, my comment is not so useful but the correct sentences are:
- We have adjust**ed**...
- ...and ha**s** props `title`